doc/en/cache.txt

```diff
@@ -0,0 +1,249 @@
+cache: working with cross-testrun state
+=======================================
+
+Usage
```

Comment:
Also, all these new docs should have a `.. versionadded:: 2.8` directive to them.